Remove redundant sliders

josihoppe opened the pull request:
removed the redundant sliders.

The corresponding data in digipipe/settings/heat_settings_panel.json and digipipe/settings/energy_settings_panel.json also had to be changed.

nesnoj reviewed (commented):
Thx for the changes!

Unfortunately django doesn't start in your branch saying
```
  File "/app/digiplan/map/views.py", line 51, in get_context_data
    forms.EnergyPanelForm(
  File "/app/digiplan/map/forms.py", line 42, in __init__
    self.fields = {item["name"]: item["field"] for item in self.generate_fields(parameters, additional_parameters)}
  File "/app/digiplan/map/forms.py", line 42, in <dictcomp>
    self.fields = {item["name"]: item["field"] for item in self.generate_fields(parameters, additional_parameters)}
  File "/app/digiplan/map/forms.py", line 49, in generate_fields
    if item["type"] == "slider":
KeyError: 'type'
```
Does it run on your machine?

josihoppe commented:
It only runs when the sliders are also deleted in the corresponding files in digipipe/settings/heat_settings_panel.json and digipipe/settings/energy_settings_panel.json

> Thx for the changes!
> 
> Unfortunately django doesn't start in your branch saying
> 
> ```
>   File "/app/digiplan/map/views.py", line 51, in get_context_data
>     forms.EnergyPanelForm(
>   File "/app/digiplan/map/forms.py", line 42, in __init__
>     self.fields = {item["name"]: item["field"] for item in self.generate_fields(parameters, additional_parameters)}
>   File "/app/digiplan/map/forms.py", line 42, in <dictcomp>
>     self.fields = {item["name"]: item["field"] for item in self.generate_fields(parameters, additional_parameters)}
>   File "/app/digiplan/map/forms.py", line 49, in generate_fields
>     if item["type"] == "slider":
> KeyError: 'type'
> ```
> 
> Does it run on your machine?

It only runs when the sliders are also deleted in the corresponding files in digipipe/settings/heat_settings_panel.json and digipipe/settings/energy_settings_panel.json

nesnoj commented:
> It only runs when the sliders are also deleted in the corresponding files in digipipe/settings/heat_settings_panel.json and digipipe/settings/energy_settings_panel.json

Ah, my brain exchanged "digipipe" with "digiplan" ;)
I'll try tomorrow..
